evm-utils/evm-bridge/src/tx_filter.rs

```diff
@@ -0,0 +1,19 @@
+use evm_rpc::Bytes;
+use evm_state::{Address, Transaction, TransactionAction};
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize)]
+pub enum TxFilter {
+    InputStartsWith(Address, Bytes),
```

Comment:
Maybe add some docs about this enum, or/and use Struct variant to add some mining to fields.
```suggestion
    InputStartsWith {
        contract: Address,
        input_prefix: Bytes
     },
```

Reply:
Done